### PR TITLE
wcloud: Allow users to specify a template for commit message

### PR DESCRIPTION
**Pull request by tomwilkie**

### CODE REVIEW

File: cmd/wcloud/cli.go [hunk 1]
```diff
@@ -29,10 +29,10 @@ var (
 
 func usage() {
 	fmt.Println(`Usage:
-	deploy <image>:<version>   Deploy image to your configured env
-	list                       List recent deployments
-	config (<filename>)        Get (or set) the configured env
-	logs <deploy>              Show lots for the given deployment`)
+	deploy (-m) <image>:<version>   Deploy image to your configured env
```
Comment (paulbellamy) [inline cmd/wcloud/cli.go]: -m probably needs a value?
isn't the commit message set in config? Or does this override it for a once-off?

Comment (paulbellamy) [inline cmd/wcloud/cli.go]: Ah, I see it is *not* set in the config. I sort of expected it to be, similar to how Slack notifications are set.

Setting it on deploy is probably fine for now, but should document `(-m "commit_message")`

**Comment by paulbellamy** [comment]
Update usage string, but LGTM.

**Comment by tomwilkie** [comment]
I prefer the idea of setting it in the config actually; it was a bit clunky this way.

**Pull request closed without merging**